game/glutil.hh

```diff
@@ -29,7 +29,7 @@ namespace glutil {
 		
 		static GLsizeiptr size() { return sizeof(shaderMatrices); };
 		static GLintptr offset() { return 0; };
-		shaderMatrices() {};
+		shaderMatrices(): projMatrix(0), mvMatrix(64), normalMatrix(128), colorMatrix(192) {};
```

Comment:
These are wrong, I think. The numbers in those comments aren't actually sizes but positions offsets.